tools/depends/target/dav1d/Makefile

```diff
@@ -0,0 +1,52 @@
+include ../../Makefile.include
+include DAV1D-VERSION
+DEPS = ../../Makefile.include Makefile
+
+# configuration settings
+CONFIGURE = $(NATIVEPREFIX)/bin/python3 $(NATIVEPREFIX)/bin/meson \
+	--buildtype=release \
+	--prefix=$(PREFIX) \
```

Comment:
I think prefix can be moved out to the cross file. Probably best to move all we can to the cross file in case any other dependency ends up using meson at some point.

```
[paths]
prefix = '$PREFIX'
```

reference https://github.com/mesonbuild/meson/blob/master/docs/markdown/Cross-compilation.md